src/Emulator/Peripherals/Peripherals/Timers/MPFS_RTC.cs

```diff
@@ -15,9 +15,9 @@
 
 namespace Antmicro.Renode.Peripherals.Timers
 {
-    public class MPFS_RTC : BasicDoubleWordPeripheral, IKnownSize
+    public class MPFS_RTC_NEW : BasicDoubleWordPeripheral, IKnownSize
```

Comment:
Please drop the `_NEW` suffix, as it's a leftover

Reply:
Dropped suffix 👍